### PR TITLE
fixed the layout option logic

### PR DESCRIPTION
**Pull request by adamrocker**
Fixed the logic of the layout option.
If there are no default option keys, the optional keys MUST NOT show anywhere because of no replaceable keys.
In other words, the layout definition MUST have a default option if it provides layout options.
スクリーンショット 2021-09-21 13 49 50

### CODE REVIEW

File: src/models/KeyboardModel.ts [hunk 1]
```diff
@@ -156,6 +156,10 @@ class KeymapItem {
     this._curr.minus(x, y);
   }
 
+  hideKey() {
+    this.op.d = true;
+  }
```
Comment (yoichiro) [inline src/models/KeyboardModel.ts:161]: @adamrocker Should we think some side effect against we use the way to hide keys with the `d` flag? If we would want to show  a string with the `d` key in the future, the logic of this time will conflict for the `d` behavior.

**Review by yoichiro — CHANGES_REQUESTED** [review]
@adamrocker I posted one comment. Could you confirm that?

File: src/models/KeyboardModel.ts (hunk 1 above)
Comment (adamrocker) [reply src/models/KeyboardModel.ts:161]: @yoichiro Agree with you. I'll delete the keys not hide them.

Comment (adamrocker) [reply src/models/KeyboardModel.ts]: @yoichiro I've fixed this in [this commit](https://github.com/remap-keys/remap/pull/643/commits/a89c29bae125bf4a03dd36c895efb45e0878c639). Could you please review it again?

**Review by yoichiro — APPROVED** [review]
@adamrocker LGTM! Thanks!

**Pull request merged**